src/docker.rs

```diff
@@ -133,38 +168,28 @@ pub fn run(
         // flag forwards the value from the parent shell
         docker.args(&["-e", var]);
     }
-    let mut env_volumes = false;
+    let mut mount_volumes = false;
+    // FIXME(emilgardis 2022-04-07): This is a fallback so that if it's hard for us to do mounting logic, make it simple(r)
+    // Preferably we would not have to do this.
+    if cwd.strip_prefix(&metadata.workspace_root).is_err() {
+        mount_volumes = true;
+    }
+
     for ref var in config.env_volumes(target)? {
         validate_env_var(var)?;
```

Comment:
I don't believe this fixes the `VAR=VALUE` issue mentioned in #743.

Reply:
i never said it would? thats #682

Reply:
Ok ok my bad, I'll mark this as resolved then.